content/docs/resiliency/uninstallation.md

```diff
@@ -10,4 +10,10 @@ This section outlines the uninstallation steps for Container Storage Modules (CS
 
 ## Uninstalling the sidecar in the CSI Driver
 
-To uninstall the sidecar in the CSI Driver, [uninstall](../../csidriver/uninstall) the driver and [reinstall](../../deployment) the driver with the `podmon` feature disabled.
\ No newline at end of file
+To uninstall the sidecar in the CSI Driver, the following steps are required.
+
+**Steps**
+1. Uninstall the driver
```

Comment:
If a customer doesn't want to completely uninstall the driver, they can also disable podmon in their values file and upgrade the driver.